analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestClassShouldHaveTestMethodTest.cs

```diff
@@ -31,32 +31,43 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class TestClassShouldHaveTestMethodTest
     {
+        private readonly VerifierBuilder verifierBuilder = new VerifierBuilder<TestClassShouldHaveTestMethod>();
```

Comment:
For simplicity, like in other tests
```suggestion
        private readonly VerifierBuilder builder = new VerifierBuilder<TestClassShouldHaveTestMethod>();
```